src/main/java/org/openstreetmap/atlas/checks/flag/FlaggedRelation.java

```diff
@@ -64,10 +67,15 @@ public FlaggedRelation(final Relation relation, final MultiPolygon geoJsonGeomet
     public JsonObject asGeoJsonFeature(final String flagIdentifier)
     {
         final JsonObject featureProperties = this.relation.getGeoJsonProperties();
+        final JsonElement osmIdentifier = featureProperties.get(OSM_IDENTIFIER_TAG);
+        final JsonElement identifier = featureProperties.get(IDENTIFIER);
+        featureProperties.remove(OSM_IDENTIFIER_TAG);
```

Comment:
Do you mind adding a comment here on why we're deleting and adding the same properties/values ?

Reply:
@danielduhh I have added the comment.